src/meta/mod.rs

```diff
@@ -56,8 +56,14 @@ impl Meta {
             println!("cannot access '{}': {}", path.display(), err);
             return Ok(meta);
         }
+		let mut content = Vec::new();
+
+		let current_meta = Self::from_path(&PathBuf::from("."))?;
+		let parent_meta = Self::from_path(&PathBuf::from(".."))?;
+
+		content.push(current_meta);
```

Comment:
We should check for the flag `-a` ie `-all` to decide if we should be showing `.` and `..`

Reply:
also maybe the creation and insert into a single line and push based on the condition.